yarn.lock

```diff
@@ -19777,7 +19777,7 @@ __metadata:
     "@testing-library/user-event": 13.5.0
     "@types/jest": 26.0.24
     "@types/loadable__component": 5.13.4
-    "@types/ramda": 0.28.15
+    "@types/ramda": ^0.28.15
```

Comment:
Should that have the caret there? See that others don’t and stick to a specific version

Reply:
Yep, you are right. I will manually delete that.